base/applications/regedit/resource.h

```diff
@@ -73,48 +72,19 @@
 
 #define ID_HELP_HELPTOPICS            32794
 #define ID_HELP_ABOUT                 32795
-#define ID_WINDOW_CASCADE             32797
-#define ID_WINDOW_TILE                32798
-#define ID_WINDOW_ARRANGEICONS        32799
-#define ID_OPTIONS_FONT               32800
-#define ID_OPTIONS_AUTOREFRESH        32801
-#define ID_OPTIONS_READONLYMODE       32802
-#define ID_OPTIONS_CONFIRMONDELETE    32803
-#define ID_OPTIONS_SAVESETTINGSONEXIT 32804
-#define ID_SECURITY_PERMISSIONS       32805
-#define ID_VIEW_TREEANDDATA           32806
-#define ID_VIEW_TREEONLY              32807
-#define ID_VIEW_DATAONLY              32808
-#define ID_VIEW_DISPLAYBINARYDATA     32810
```

Comment:
Ok, but wouldn't it make more sense to readd a fresh ID then? Who will remember that ancient ID when he will be going to work on that? It is much more likely that people will introduce some new ID then because they are not aware of the old corpse, isn't it?

Reply:
You have a point, so let's remove the ID for now.